### DISCUSSION TITLE
[v3] Property not found on component after reset()

### DISCUSSION THREAD

**Discussion by devront-it** [livewire/livewire#6540] [7. Bugs (archived)]
### Livewire version

v3.0.1

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Create a component with a property like ``public $test_property = null;``
2. Create a button that calls a component method:
```php
public function test() {
    $this->reset('test_property');
    $this->test_property;
}
```
3. This will throw "Property $test_property not found on Component..."

### Additional information

Everything works fine when the $test_property is declared with a value other than null (e.g. empty string).

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Simoneu01** [comment]
I have the same issue on my application, I've created a failing test for this error: https://github.com/livewire/livewire/pull/6729

**Comment by edalzell** [comment]
Likely related to https://github.com/livewire/livewire/pull/6345
